packages/synapse-react-client/src/components/TanStackTable/ColumnHeaderEnumFilter.test.tsx

```diff
@@ -14,52 +17,89 @@ type TestTableRow = {
   multiSelectWithCounts: string
   singleSelect: string
   numeric: number
+  defaultClientSideFacetCalculation: string
 }
 
 const columnHelper = createColumnHelper<TestTableRow>()
 
+function getFacetedUniqueValues(table: Table<TestTableRow>, columnId: string) {
+  if (columnId === 'multiSelect') {
+    return () =>
+      new Map([
+        ['foo', 0],
+        ['bar', 0],
```

Comment:
TanStack table does not provide an API for facets without counts, so I added some bespoke logic to deal with this when all of the counts are 0. The TanStack table types do not allow this value to be null.

Reply:
It's a shame that the table type does not allow `undefined` or `null`